src/pages/p2p/components/_numbers.js

```diff
@@ -8,11 +8,11 @@ import device from 'themes/device'
 
 const content = [
     {
-        header: <Localize translate_text="2000+" />,
+        header: <Localize translate_text="4000" />,
```

Comment:
I'm just curious is there a translation for numbers 😄

Reply:
Good question, I mean there is but I don't know for this instance whether they would want to translate it to non-number figures 😂

Reply:
I see, makes sense 😄